src/ucm/util/reloc.c

```diff
@@ -312,12 +312,52 @@ static ucs_status_t ucm_reloc_apply_patch(ucm_reloc_patch_t *patch,
     return ctx.status;
 }
 
+/* read serinfo from 'module_path', result buffer must be destroyed
+ * by free() call */
+static Dl_serinfo *ucm_dlopen_load_serinfo(const char *module_path)
+{
+    Dl_serinfo *serinfo = NULL;
+    Dl_serinfo stub;
+    void *caller_module;
+    int res;
+
+    caller_module = ucm_reloc_orig_dlopen(module_path, RTLD_NOW);
```

Comment:
maybe use RTLD_LAZY?

Reply:
actually module should be completely initialized at this moment (it is calling dependency function)
ok, fixed

Reply:
RTLD_NOW also resolves all symbols, it doesn't necessarily happen